api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/util/Wrappers.java

```diff
@@ -75,4 +79,20 @@ public static void setupTenantRoute(Router router, HttpMethod method, String pat
                 },
                 ctx.response()::end);
     }
+
+    public static Transformer<String, String> toJsonArray(RoutingContext ctx) {
```

Comment:
No no, nothing like this. Real JSON parser (like the example I gave you). This one tests nothing useful for us.